packages/core/src/model.d.ts

```diff
@@ -382,6 +382,8 @@ export interface WhereOperators<AttributeType = any> {
   [Op.contains]?:
     // RANGE @> ELEMENT
     | AttributeType extends Range<infer RangeType> ? OperatorValues<OperatorValues<NonNullable<RangeType>>> : never
+    // jsonb @> ELEMENT
+    | AttributeType extends object ? OperatorValues<Partial<AttributeType>> : never
```

Comment:
Instead of using `|`, this condition could be merged with the above one

```ts
// RANGE @> ELEMENT
| AttributeType extends Range<infer RangeType> ? OperatorValues<OperatorValues<NonNullable<RangeType>>> 
  // jsonb @> ELEMENT
  : AttributeType extends object ? OperatorValues<Partial<AttributeType>> 
  : never
```